cmd/buildah/common.go

```diff
@@ -158,3 +159,17 @@ func getDockerAuth(creds string) (*types.DockerAuthConfig, error) {
 		Password: password,
 	}, nil
 }
+
+func validateFlags(c *cli.Context, flags []cli.Flag) error {
+	for _, flag := range flags {
+		if reflect.TypeOf(flag).String() == "cli.StringFlag" {
+			f := flag.(cli.StringFlag)
```

Comment:
maybe it's late and I'm being confused, but I'm not seeing a check for the actual value?  It looks like you're checking the name and failing if it begins with a dash.  Can you tell me what value is expected in 'f.Name' and 'name' if it checks 'buildah pull' with these params "--creds=myusername:mypassword" and  "--cert-dir ~/auth"?

Reply:
That type of check would happen per option. I am just looking for the case where a user forgot to enter a value fro a string field.

--format --foobar  
Should fail since --format was expecting a value but instead the CLI handed it a "option".